config/locales/errors/en.yml

```diff
@@ -9,12 +9,13 @@ en:
     messages:
       already_confirmed: was already confirmed, please try signing in
       blank: Please fill in this field.
+      confirmation_code_incorrect: Incorrect code. Did you type it in correctly?
```

Comment:
Are we explicitly making a distinction between the USPS confirmation code and other security codes? For example, we have `otp_incorrect: Security code is incorrect`, but here we are changing the language. Is that intentional?